rclcpp/include/rclcpp/loaned_message.hpp

```diff
@@ -202,6 +191,29 @@ class LoanedMessage
 
   /// Deleted copy constructor to preserve memory integrity.
   LoanedMessage(const LoanedMessage<MessageT> & other) = delete;
+
+private:
+  static void DestroyMessage(
```

Comment:
This should be `destroy_message()` to follow the style of the package.

```suggestion
  static void destroy_message(
```